sys/nginx/lua/hook.lua

```diff
@@ -1,5 +1,17 @@
-local regex = [[/(circleci|circleci/.+|circleci/.+\.svg|circleci\.svg|composerlock|composerlock\.svg|d/daily|d/daily\.png|d/daily\.svg|d/monthly|d/monthly\.png|d/monthly\.svg|d/total|d/total\.png|d/total\.svg|dependents|dependents\.svg|downloads|downloads\.png|downloads\.svg|gitattributes|gitattributes\.svg|license|license\.png|license\.svg|suggesters|suggesters\.svg|v/stable|v/stable\.png|v/stable\.svg|v/unstable|v/unstable\.png|v/unstable\.svg|version|version\.png|version\.svg)$]]
+local regex = [[/(]]
+regex = regex .. [[circleci(/.+)?(\.svg)?]]
+regex = regex .. [[|composerlock(\.svg)?]]
```

Comment:
I mean it should work something like this:

```
regex = [[/(
circleci(?:/.+)?(?:\.svg)?
|composerlock(?:\.svg)?
|d/(?:daily|monthly|total)(?:\.svg|\.png)?
...
...
)$]]
```

Reply:
in that way don't we include the LF in the middle of the regex? I guess that will break the matching.

Reply:
well .. I suppose it is automatically removed, provided that `[[ ]]` is a multiline string syntax .. I supposed it would behave like the `|` in YAML.

The lua doc states:

> Literal strings can be delimited by matching single or double quotes, and can contain the following C-like escape sequences: '\a' (bell), '\b' (backspace), '\f' (form feed), '\n' (newline), '\r' (carriage return), '\t' (horizontal tab), '\v' (vertical tab), '\\' (backslash), '\"' (quotation mark [double quote]), and '\'' (apostrophe [single quote]). **Moreover, a backslash followed by a real newline results in a newline in the string.**

So I guess it should work .. have you already tried to print such a kind of string to have a real world test case?

Reply:
No, you are right .. it would retain the new line .. but using a _literal string_ should do the trick:

```
regex = '/(
circleci(?:/.+)?(?:\.svg)?
|composerlock(?:\.svg)?
|d/(?:daily|monthly|total)(?:\.svg|\.png)?
...
...
)$'
```